### ISSUE TITLE
Calling maniphest.edit with transaction type that contains dot(.)

### ISSUE THREAD

**Issue by ottob** [disqus/python-phabricator#31]
How can I call "projects.add" of https://secure.phabricator.com/conduit/method/maniphest.edit/ ?

I've tried:
```
phab.maniphest.edit(id=taskId, projects.add=['PHID-PROJ-blahblah'])
```
then I get:
```
File "jira.py", line 62
    phab.maniphest.edit(id=taskId, projects.add=['PHID-PROJ-blahblah'])
SyntaxError: keyword can't be an expression
```
and I've tried:
```
dct = {
	'id': taskId,
	'projects.add': ['PHID-PROJ-zi7t6ztcntlder4ihcpr']
}
phab.maniphest.edit(**dct)
```
but then I get:
```
File "jira.py", line 66, in task_setproject
    phab.maniphest.edit(**dct)
  File "/Library/Python/2.7/site-packages/phabricator/__init__.py", line 235, in __call__
    return self._request(**kwargs)
  File "/Library/Python/2.7/site-packages/phabricator/__init__.py", line 253, in _request
    raise ValueError('Missing required argument: %s' % key)
ValueError: Missing required argument: transactions
```

Any clues?

**Comment by ottob** [comment]
I got it to work by doing this:
```
phab.maniphest.edit(objectIdentifier=task, transactions=[{'type': 'projects.add', 'value': ['PHID-PROJ-blahblah']}])
```
and then commenting out these lines: https://github.com/disqus/python-phabricator/blob/master/phabricator/__init__.py#L254-L259

**Comment by BYK** [comment]
I don't think you need to change the validation after the way you changed your code. Did you try it without commenting out validation?

**Comment by BYK** [comment]
I think this should be fixed with #40 or #41. Reopen if it did not.
